src/event/ngx_event_openssl.h

```diff
@@ -199,6 +199,14 @@ typedef struct {
 #define NGX_SSL_CACHE_CA    3
 
 
+#ifdef SSL_OP_NO_TLSv1_2
+#define NGX_SSL_PROTOCOLS   NGX_SSL_TLSv1_2|NGX_SSL_TLSv1_3
+#else
+#define NGX_SSL_PROTOCOLS   NGX_SSL_TLSv1_2|NGX_SSL_TLSv1_3                   \
+                            |NGX_SSL_TLSv1|NGX_SSL_TLSv1_1
```

Comment:
For safety, the value needs to be enclosed in parentheses. Even though we know where this value goes now, but this may change in the future and result in an unexpected error.